From b758e8f97ebd6d380b741ee498a842fe6bc3be9c Mon Sep 17 00:00:00 2001
From: Nathan Chancellor <natechancellor@gmail.com>
Date: Tue, 25 Feb 2020 13:02:06 -0700
Subject: [PATCH 19/23] Revert "x86/boot/compressed: Remove unnecessary
 sections from bzImage"

Replacing with the latest version from tip's x86/boot branch.

This reverts commit e11831d0ada3efc7f9268f35c80257f7e1b1dd0c.

Signed-off-by: Nathan Chancellor <natechancellor@gmail.com>
---
 arch/x86/boot/compressed/vmlinux.lds.S | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/arch/x86/boot/compressed/vmlinux.lds.S b/arch/x86/boot/compressed/vmlinux.lds.S
index 12a20603d92e..508cfa6828c5 100644
--- a/arch/x86/boot/compressed/vmlinux.lds.S
+++ b/arch/x86/boot/compressed/vmlinux.lds.S
@@ -73,9 +73,4 @@ SECTIONS
 #endif
 	. = ALIGN(PAGE_SIZE);	/* keep ZO size page aligned */
 	_end = .;
-
-	/* Discard all remaining sections */
-	/DISCARD/ : {
-		*(*)
-	}
 }
-- 
2.25.1

